python/rpdk/go/codegen.py

```diff
@@ -169,7 +175,9 @@ def generate(self, project):
         path = root / "main.go"
         LOG.debug("Writing project: %s", path)
         template = self.env.get_template("main.go.tple")
-        importpath = Path(project.settings["import_path"])
+        importpath = Path(
+            project.settings.get("import_path", project.settings.get("importpath"))
```

Comment:
with this in place there seems no need to have all the other `import_path` -> `importpath` changes